src/components/calcButton.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import './calculator.css';
+
+class CalcButton extends React.Component {
+  render() {
+    return (
+      <div className="bordeer">
+        <div className="maingrid">
+          <button type="button" className="btn1">0</button>
+          <button type="button" className="btn2">AC</button>
+          <button type="button" className="btn3">+/-</button>
+          <button type="button" className="btn4">%</button>
+          <button type="button" className="btn5">÷</button>
+          <button type="button" className="btn6">7</button>
+          <button type="button" className="btn7">8</button>
+          <button type="button" className="btn8">9</button>
+          <button type="button" className="btn9">x</button>
+          <button type="button" className="btn10">4</button>
+          <button type="button" className="btn11">5</button>
+          <button type="button" className="btn12">6</button>
+          <button type="button" className="btn13">-</button>
+          <button type="button" className="btn14">1</button>
+          <button type="button" className="btn15">2</button>
+          <button type="button" className="btn16">3</button>
+          <button type="button" className="btn17">+</button>
```

Comment:
- [ ] Kindly note that it is advised in the project requirements that you use functional components.  One good reason for using functional components in React is that they are more performant as  React has optimized the handling of functional components
- [ ] Also, a mentioned earlier, Kindly ensure the calculator component has at least one child component. Hint: you can create a component that renders buttons

Reply:
done thanks